### ISSUE TITLE
Simplify power actor to rely on cron invoked every epoch

### ISSUE THREAD

**Issue by anorth** [filecoin-project/specs-actors#906]
We can now rely on cron being invoked for every tipset, including nulls. Simplify the power actor tracking of last processed epoch to rely on this. Be careful to retain the ability to schedule things for epochs in the past.

 The market actor may enjoy a similar

**Comment by ZenGround0** [comment]
On a related note @davidad I want to check that the current state of things is ok given that we now run an update every epoch, null and non null.  It appears this might contradict this suggestion from the Alpha Beta Filtering spec:

> this revision algorithm should not be run on behalf of null rounds while catching up; instead, dt should be set to 1 + [# of consecutive null rounds] when running on the next non-null round.

**Comment by davidad** [comment]
@ZenGround0, thanks for catching & flagging. The concern with updating the filter on null rounds is that it looks like the power velocity is zero since no power can be added in a null round. In general when trying to estimate the state and trajectory of a system, it helps not to confuse missing data for actual zeros. But I think this won’t make a big difference, assuming the length of consecutive-null-round sequences is always negligibly short compared to the filter's time-constant (which, with current tuning, is about 1080 epochs). What's the longest consecutive-null-round count we've seen on an otherwise-healthy testnet?

**Comment by ZenGround0** [comment]
So far longest run we've tracked with the current incarnation of the protocol is 36.  @placer14 provided the following data from recent calibration net:

```
height	consecutive_null_rounds
1861	36
2197	32
2286	22
2034	15
2090	13
1985	12
2108	11
1906	11
2245	8
1936	6
2096	3
2251	3
7022	2
3579	2
3691	2
```

**Comment by placer14** [comment]
FWIW, that data was sourced via lotus running at (7c775390) (which precedes published tag `ntwk-calibration-7.29.0`).

**Comment by davidad** [comment]
That’s more than I expected, but I think it’s okay. Should affect the filter estimates by less than 2%.
